composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "powderblue/sf-continent-country-type",
+    "type": "symfony-bundle",
+    "authors": [
+        {
+            "name": "Adrian Dumitrache",
+            "email": "adrian.dumitrache@trisoft.ro"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "PowderBlue\\SfContinentCountryTypeBundle\\": ""
+        }
+    },
+    "require": {
+        "symfony/symfony": "2.8.*"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.3",
+        "squizlabs/php_codesniffer": "^2.6"
+    },
+    "scripts": {
+        "dev:code:check-quality": [
```

Comment:
I would rename "dev:code" to just "app" because there won't be runtime modes.  Thanks.